src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx

```diff
@@ -179,44 +180,36 @@ export const PupilViewsLessonOverview = ({
             <OakFlex $gap="space-between-s" $flexDirection="column">
               {lessonReviewSections.includes("intro") && (
                 <OakLessonNavItem
-                  as="button"
+                  {...getSectionLinkProps("intro", updateCurrentSection)}
                   lessonSectionName="intro"
-                  onClick={() => updateCurrentSection("intro")}
                   progress={pickProgressForSection("intro")}
-                  disabled={!isMounted}
```

Comment:
Links are no longer disabled since the user can now navigate before the page has hydrated.